### PR TITLE
[quant] Expose supportedQEngines to python

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26591 [quant] Add QuantFusion to graph_executor
* **#26474 [quant] Expose supportedQEngines to python**

Summary:
att

Test Plan:
python test/test_torch.py

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17517373](https://our.internmc.facebook.com/intern/diff/D17517373)

### CODE REVIEW

File: test/test_torch.py [hunk 1]
```diff
@@ -2101,6 +2101,15 @@ def test_inference(default_dtype):
         test_inference(torch.float64)
         test_inference(torch.float32)
 
+    def test_qengnie(self):
+        qengines = torch._C._supported_qengines()
+        # [TODO] Enable after the interface change
+        # original_qe = torch._C._get_qengine()
+        # for qe in qengines:
```
Comment (supriyar) [inline test/test_torch.py]: `qe` is an int but set_qengine expects `at::QEngine`. Will this work?

Comment (jerryzh168) [reply test/test_torch.py]: it doens't work right now that's why I commented them out. it should work after the interface change. we should be returning/pass `int` in `_get_qengine` and `_set_qengine` as well

**Review by supriyar — APPROVED** [review]
Looks good. 
The plan after this is to change qengine to string (user facing) and return an int from `supported_qengines`.

**Comment by jerryzh168** [comment]
plan is to change `_set_qengine` and `_get_qengine` to return/receive int. `supported_qengines` already returns a list of int

**Pull request closed without merging**